src/containers/unrolledlist.d

```diff
@@ -565,7 +565,7 @@ private:
 			static if (BookkeepingType.sizeof < uint.sizeof)
 				immutable uint notReg = ~(cast(uint) registry);
 			else
-				immutable uint notReg = ~registry;
+				immutable uint notReg = cast(uint) (~registry);
```

Comment:
@Hackerpilot , to answer to a previous review comment (#137), the two branches are not the same. The first is to fix the deprecation message related to integral promotion not done, the other fixes the issue.